pkg/kobject/kobject.go

```diff
@@ -92,6 +92,7 @@ type ServiceConfig struct {
 	MemLimit        yaml.MemStringorInt `compose:"mem_limit" bundle:""`
 	TmpFs           []string            `compose:"tmpfs" bundle:""`
 	Dockerfile      string              `compose:"dockerfile" bundle:""`
+	Vols            []Volumes           `compose:"" bundle:""`
```

Comment:
I think it is not a good idea to have both `Volumes` and `Vols`, please remove the `[]String`